lib/opentelemetry_plug.ex

```diff
@@ -100,42 +99,34 @@ defmodule OpentelemetryPlug do
 
   @doc false
   def handle_stop(_, _measurements, %{conn: conn}, _config) do
-    if in_span?() do
-      Tracer.set_attribute("http.status_code", conn.status)
-      # For HTTP status codes in the 4xx and 5xx ranges, as well as any other
-      # code the client failed to interpret, status MUST be set to Error.
-      #
-      # Don't set the span status description if the reason can be inferred from
-      # http.status_code.
-      if conn.status >= 400 do
-        Tracer.set_status(OpenTelemetry.status(:error, ""))
-      end
-
-      Tracer.end_span()
-      restore_parent_ctx()
```

Comment:
You do still need to manage the saving and restoring of the span ctx. That isn't done automatically anymore, so you had it right the first time. The `in_span?` deal is a relic from pre-ctx days

Reply:
Woops.